CONSOLE-4492,CONSOLE-4514: Align spacing of page elements with PatternFly and remove border after PageHeading

rhamilto opened the pull request:
Includes changes from https://github.com/openshift/console/pull/14805, which will be closed so we can get all the changes in in light of CI flakiness.

After

![localhost_9000_k8s_all-namespaces_import](https://github.com/user-attachments/assets/cf4d59cd-617d-4507-bd81-8e0d8d23540e)
![localhost_9000_k8s_all-namespaces_import (1)](https://github.com/user-attachments/assets/6daabef1-c1dd-410a-98f4-940535beca03)
![localhost_9000_k8s_all-namespaces_import (2)](https://github.com/user-attachments/assets/1a13eb16-936f-4426-9e20-0870c2b21cfb)
![localhost_9000_k8s_all-namespaces_import (3)](https://github.com/user-attachments/assets/89972592-dfb2-48cf-a80e-e7661d7590bb)
![localhost_9000_k8s_all-namespaces_import (4)](https://github.com/user-attachments/assets/decf235c-8cb8-4427-afa6-c5cc6f93eae3)

rhamilto commented:
/assign @logonoff
/assign @yapei 

Follow-on to previously approved story, so assigning approvals:
/label px-approved
/label docs-approved

logonoff reviewed (approved):
changes look good to me. I'll give `lgtm` once https://github.com/openshift/console/pull/14805 merges as I anticipate that a few more rebases will probably still be needed lol

XiyunZhao commented:
I have checked all regular pages that include the Tab menu bar, the border has been successfully removed, and no omissions were found
/label qe-approved

logonoff commented:
/label acknowledge-critical-fixes-only

logonoff reviewed (approved):
/lgtm

logonoff commented:
/label tide/merge-method-squash acknowledge-critical-fixes-only 
/lgtm

logonoff commented:
/retitle CONSOLE-4492, CONSOLE-4514: remove border after PageHeading, above HorizontalNav

logonoff commented:
/unhold

logonoff commented:
/retitle CONSOLE-4492, CONSOLE-4514: remove border after PageHeading, above HorizontalNav

logonoff commented:
/label acknowledge-critical-fixes-only

logonoff commented:
/label tide/merge-method-squash

logonoff commented:
/retitle CONSOLE-4492,CONSOLE-4514: Align spacing of page elements with PatternFly and remove border after PageHeading

logonoff commented:
/retitle CONSOLE-4492,CONSOLE-4514: Align spacing of page elements with PatternFly and remove border after PageHeading

rhamilto commented:
/retest

logonoff commented:
/retest

rhamilto commented:
/retest

rhamilto commented:
/retest

rhamilto commented:
/retest

rhamilto commented:
/retest

rhamilto commented:
Something in this PR is making the tests run much more slowly. The two successful runs took over 6 hours. A typical successful run takes around 4.5 hours. Need to investigate. 

/hold

rhamilto commented:
/hold cancel

logonoff commented:
/lgtm

logonoff commented:
/lgtm

logonoff commented:
/lgtm cancel

there appear to be some breaking changes with updating cypress

rhamilto commented:
Added potential Cypress upgrade fix and rebased.  If the Cypress fix works, we'll want a follow-on story to remove `injectDocumentDomain` from the Cypress configs as that option will eventually be removed.  See https://docs.cypress.io/app/references/changelog#14-0-0

logonoff commented:
/lgtm

rhamilto commented:
> /lgtm

Thank you, @logonoff!

rhamilto commented:
> injectDocumentDomain

https://issues.redhat.com/browse/CONSOLE-4537

logonoff commented:
/honk
